L1Trigger/L1TTrackMatch/plugins/L1TrackJetProducer.cc

```diff
@@ -63,73 +70,72 @@ class L1TrackJetProducer : public stream::EDProducer<> {
 
   // ----------member data ---------------------------
 
+  edm::ESGetToken<TrackerTopology, TrackerTopologyRcd> tTopoToken_;
+  const EDGetTokenT<vector<TTTrack<Ref_Phase2TrackerDigi_>>> trackToken_;
+  const EDGetTokenT<l1t::VertexWordCollection> PVtxToken_;
   vector<Ptr<L1TTTrackType>> L1TrkPtrs_;
   vector<int> tdtrk_;
-  const float trkZMax_;
-  const float trkPtMax_;
-  const float trkPtMin_;
-  const float trkEtaMax_;
-  const float nStubs4PromptChi2_;
-  const float nStubs5PromptChi2_;
-  const float nStubs4PromptBend_;
-  const float nStubs5PromptBend_;
-  const int trkNPSStubMin_;
-  const int lowpTJetMinTrackMultiplicity_;
-  const float lowpTJetThreshold_;
-  const int highpTJetMinTrackMultiplicity_;
-  const float highpTJetThreshold_;
-  const int zBins_;
-  const int etaBins_;
-  const int phiBins_;
-  const double minTrkJetpT_;
+  float trkZMax_;
```

Comment:
Why did you remove all `const` qualifiers here?
Please put them back!

Reply:
they have been added